orca-queue-redis/src/test/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueueSpec.kt

```diff
@@ -50,3 +59,29 @@ private val createQueue = { clock: Clock,
 private fun shutdownCallback() {
   redis?.destroy()
 }
+
+class ConvertToMessageSpec : Spek({
+  describe("should support deserializing a nested message") {
```

Comment:
I bet @robfletcher could tell me how to add this only to the `RedisQueueSpec` (that's extending from `QueueSpec`).